.env-cmdrc

```diff
@@ -101,5 +101,8 @@
   "punchout": {
     "CX_PUNCHOUT": "true",
     "CX_B2B": "true"
+  },
+  "newAuthFlow": {
+    "CX_NEW_AUTH_FLOW": "true"
```

Comment:
This environment variable is unused.  What is the intention for it?

Reply:
sorry missed that one